### ISSUE TITLE
NSX Edge LB Pool Port

### ISSUE THREAD

**Issue by pandom** [vmware/powernsx#38]
An NSX Load Balancer pool can have servers listening on numerous ports.

(Security Server in Horizon can listen on 443, 8443, 4172). Creating a new pool in PowerNSX dictates a value in Port.

Port should be changed to a non mandatory attribute for New-Pool creation.

**Comment by pandom** [comment]
Line 20654 - new-nsxloadbalancermemberspec

```
       [Parameter (Mandatory=$true)]
            [ValidateRange(1,65535)]
            [int]$Port,   
```
should be

```
       [Parameter (Mandatory=$false)]
            [ValidateRange(1,65535)]
            [int]$Port,   
```

Also update the validate check for this when adding to Pool.

**Comment by nmbradford** [comment]
Fixed in #71
